### ISSUE TITLE
Open GearBrowser via ApplicationControl

### ISSUE THREAD

**Issue by NickHatBoecker** [fin10/GearBrowser#5]
Hey there,

I'm trying to implement some oAuth2 flow in my wearable web app.
"http://tizen.org/appcontrol/operation/view" is not working with my URL so I would like to open your GearBrowser in order to load the url.
Is this already possible or could you provide some code or the right place to start in your existing code, so I could try to fork and implement it myself?

Many thanks!

Best regards
Nick

**Comment by Changyong-Jeon** [comment]
Good idea! If you work it and want to merge it to this project, I'll get it and deploy this app with your new feature.

**Comment by fin10** [comment]
Good idea! If you work it and want to merge it to this project, I'll get it and deploy this app with your new feature.

**Comment by NickHatBoecker** [comment]
@fin10 Unfortunately I'm a total C noob, but you gladly already implemented ```app_control``` function and an event listener. So I think I can get the URL there.

Can you tell me how to programmatically open a url from here once I got it?
https://github.com/fin10/GearBrowser/blob/master/GearBrowser/src/gearbrowser.c#L85

**Comment by fin10** [comment]
How about checking [this guide](https://developer.tizen.org/development/guides/native-application/application-management/application-controls?langredirect=1#export_appcontrol)? It's how to define the app control for other applications.
And [this sample](https://developer.tizen.org/development/sample/native/AppFW/LaunchOnEventServiceApp) will help you to understand it more.

**Comment by NickHatBoecker** [comment]
@fin10 The Application Control ist not the problem. But I need to understand your code, that's why I wanted to know if you could tell me how to render a url in your browser from the code part I mentioned above. Like what's the start, when I managed to get a url via Application Control.

**Comment by fin10** [comment]
If you so, I think you should extend arguments for specific url on `web_layout_open` in [this line](https://github.com/fin10/GearBrowser/blob/master/GearBrowser/src/gearbrowser.c#L66). And it should be better to check [this reference guide](https://developer.tizen.org/development/api-references/native-application?redirect=https://developer.tizen.org/dev-guide/2.3.1/org.tizen.native.wearable.apireference/group__WEBVIEW.html) if you have to get a response of oauth login.
